WEATHERAPP/Components/Login/LoginPage.css

```diff
@@ -0,0 +1,112 @@
+body
+{
+   margin: 0;
+}
+.background
+{
+   height: 100vh;
+   background-image:url("../../Assets/L&D-BG\ 4.png");
+   display: flex;
+   flex-direction: column;
+   justify-content: space-evenly;
+}
+.app-header
+{
+   text-align: center;
+   font:700 72px system-ui;
+}
+.main-contents
+{
+   display: inline-flex;
+   justify-content: space-around;
+}
+.weather-image
+{
+   width: 300px;
+   height: 300px;
+}
+.login-form
+{
+   display: flex;
+   flex-direction: column;
+   align-items: center;
+   justify-content: space-evenly;
+   background-color: rgba(132, 95, 95, 1);
+   width:300px;
+   height:300px;
+}
+
+.input-section
+{
+   display: flex;
+   flex-direction: column;
+}
+.input-label
+{
+   color: rgba(255, 255, 255, 1);
+   font-family: sans-serif;
+   font-weight: 100;
+}
+.input-fields
+{
+   width: 200px;
+   height: 30px;
+   border-radius: 6px;
+}
+.buttons-section
+{
+   width: 200px;
+   display: inline-flex;
+   justify-content: space-between;
+}
+.navigating-button
+{
+  height: 30px;
+  width: 90px;
+  border-radius:5px ;
+}
+#sign-in
+{
+   background-color: rgba(255, 255, 255, 1);
+}
+#sign-up
+{
+   background-color: rgba(44, 44, 44, 1);
+   color: rgba(255, 255, 255, 1);
+}
+
+@media (max-width: 600px) {
+  .app-header {
+     font-size: 36px;
+  }
+  
+  .main-contents {
+    display: inline-flex;
+    justify-content: space-around;
+  }
+  
+  .weather-image {
+     width: 150px;
+     height: 150px;
+  }
+  
+  .login-form {
+     width: 150px;
+     height: 150px;
+  }
+  
+  .input-fields {
+     width: 100px;
+     height: 15px;
+  }
+  
+  .buttons-section {
+     width: 100px;
+  }
+  
+  .navigating-button {
+    height: 15px;
```

Comment:
Consider adding cursor: pointer and transition properties for better user experience.

Reply:
resolved